dev-python/QtPy/QtPy-1.9.0-r4.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..8} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Abstraction layer on top of PyQt5 and PySide2 and additional custom QWidgets"
+HOMEPAGE="https://github.com/spyder-ide/qtpy"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+
+_IUSE_QT_MODULES="
+	declarative designer gui help location multimedia network opengl
+	positioning printsupport sensors serialport sql svg testlib webchannel
+	webengine websockets widgets x11extras xml xmlpatterns
+"
+IUSE="+pyqt5 pyside2 ${_IUSE_QT_MODULES}"
+
+REQUIRED_USE="|| ( pyqt5 pyside2 )"
+
+# These flags are currently *not* common to both the PySide2 and PyQt5 ebuild
+# Disable them for now, please check periodically if this is still up to date.
+# 	bluetooth? ( pyqt5 )
+# 	dbus? ( pyqt5 )
+# 	networkauth? ( pyqt5 )
+# 	webkit? ( pyqt5 )
```

Comment:
For the record, I think we don't need to be concerned about webkit as it's deprecated.